test/testhelpers/helpers.go

```diff
@@ -289,3 +290,28 @@ func ExecPodCommandWithOptions(config *rest.Config, client kubernetes.Interface,
 
 	return stdoutBuf, stderrBuf, nil
 }
+
+func GatherRegistryResources(outputPath, scriptPath, name, namespace string) error {
+	path, err := exec.LookPath(scriptPath)
```

Comment:
Just for reviewers. this `exec.LookPath` is a pretty nice utility function that I didn't realize existed before re-reading that exec package documentation. It essentially acts as a `os.Stat` operation to ensure the path to the binary or script exists, which is nice. We may want to update the rest of our instances of this package to also use this function.

Reply:
Also, the link to that function documentation: https://golang.org/pkg/os/exec/#LookPath.

Reply:
👍